schema.cc

```diff
@@ -19,6 +19,7 @@
  * along with Scylla.  If not, see <http://www.gnu.org/licenses/>.
  */
```

Comment:
changelog, what problem are you trying to solve and how are you going about it.

Reply:
done.